frontend/api/api.ts

```diff
@@ -3628,6 +3628,25 @@ export interface UserSet {
      */
     'user_id'?: string;
 }
+/**
+ * 
+ * @export
+ * @interface UtilsPreSignedURL
+ */
+export interface UtilsPreSignedURL {
```

Comment:
Why having 
```
type GetDocDownloadUrlResponse struct {
	PreSignedURL string `json:"pre_signed_url"`
	ExpiresAt    int64  `json:"expires_at"`
} //@Name GetDocDownloadUrlResponse
```

and this interface ? Seems to be redondent no ?

Reply:
I always create structs and separate the inputs and outputs (responses) for endpoints. It's also so that it appears like this in the docs.

Reply:
I can remove the interface